deploy.py

```diff
@@ -36,129 +37,137 @@ def deploy(bento_bundle_path, deployment_name, config_json):
-            resource_group_name,
-        ]
-    )
+    with console.status(f"Creating Azure ACR"):
+        run_shell_command(
+            [
+                "az",
+                "acr",
+                "create",
+                "--name",
+                acr_name,
+                "--sku",
+                azure_config["acr_sku"],
+                "--resource-group",
+                resource_group_name,
+            ]
+        )
 
-    # build and push docker
-    run_shell_command(
-        [
-            "az",
-            "acr",
-            "login",
-            "--name",
-            acr_name,
-            "--resource-group",
-            resource_group_name,
-        ]
-    )
+        # build and push docker
+        run_shell_command(
+            [
+                "az",
+                "acr",
+                "login",
+                "--name",
+                acr_name,
+                "--resource-group",
+                resource_group_name,
+            ]
+        )
+    console.print(f"Created Azure ACR {acr_name}")
 
     docker_image_tag = (
         f"{acr_name}.azurecr.io/{bento_metadata.name}:{bento_metadata.version}".lower()
     )
-    print(f"Build and push image {docker_image_tag}")
+
 
     major, minor, _ = bento_metadata.env.python_version.split(".")
-    build_docker_image(
-        context_path=deployable_path,
-        image_tag=docker_image_tag,
-        dockerfile="Dockerfile-azure",
-        additional_build_args={
-            "BENTOML_VERSION": LAST_PYPI_RELEASE_VERSION,
-            "PYTHON_VERSION": major + minor,
-        },
-    )
-    push_docker_image_to_repository(docker_image_tag)
 
-    print(f"Deploying Azure function {function_name}")
-    docker_username, docker_password = get_docker_login_info(
-        resource_group_name, acr_name
-    )
-    run_shell_command(
-        [
-            "az",
-            "functionapp",
-            "create",
-            "--name",
-            function_name,
-            "--storage-account",
-            storage_account_name,
-            "--resource-group",
-            resource_group_name,
-            "--plan",
-            function_plan_name,
-            "--functions-version",
-            "3",
-            "--deployment-container-image-name",
-            docker_image_tag,
-            "--docker-registry-server-user",
-            docker_username,
-            "--docker-registry-server-password",
-            docker_password,
-        ]
-    )
+    with console.status("Building image"):
+        build_docker_image(
+            context_path=deployable_path,
+            image_tag=docker_image_tag,
+            dockerfile="Dockerfile-azure",
+            additional_build_args={
+                "BENTOML_VERSION": LAST_PYPI_RELEASE_VERSION,
+                "PYTHON_VERSION": major + minor,
+            },
+        )
+        push_docker_image_to_repository(docker_image_tag)
+    # console.print(f"Build and push image {docker_image_tag}")
```

Comment:
can you make another console.status saying "Pushing image" since pushing can take a lot of time. and why is this line commented out?

Reply:
Added a console status and print for "Pushing image". Pushed commit.